test/skimNtuple_HHbtag.cpp

```diff
@@ -5366,129 +5368,152 @@ int main (int argc, char** argv)
 			    
-			  TLorentzVector tlv_HHbregrsvfit  = tlv_bH_particleNet_regression + tlv_tauH_SVFIT;
-			  theSmallTree.m_HHbregrsvfit_pt   = tlv_HHbregrsvfit.Pt();
-			  theSmallTree.m_HHbregrsvfit_eta  = tlv_HHbregrsvfit.Eta();
-			  theSmallTree.m_HHbregrsvfit_phi  = tlv_HHbregrsvfit.Phi();
-			  theSmallTree.m_HHbregrsvfit_m    = tlv_HHbregrsvfit.M();
+				  // computing the vector made of the fatjet (H-bb) + svfit  H-tt
+				  TLorentzVector tlv_bH_particleNet_regression;
+				  tlv_bH_particleNet_regression.SetPtEtaPhiM(tlv_fj.Pt(), tlv_fj.Eta(), tlv_fj.Phi(),
+															 theBigTree.ak8jets_particleNetMDJetTags_mass->at(fjIdx));
 			    
-			  std::map<std::string, std::vector<float>> result = ParticleNet_SF(tlv_fj.Pt(), PERIOD);
-			  // Set scale factors for different keys
-			  setScaleFactor(result, "HP", theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF, theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_up, theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_down);
-			  setScaleFactor(result, "MP", theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF, theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_up, theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_down);
-			  setScaleFactor(result, "LP", theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF, theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_up, theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_down);
+				  TLorentzVector tlv_HHbregrsvfit  = tlv_bH_particleNet_regression + tlv_tauH_SVFIT;
+				  theSmallTree.m_HHbregrsvfit_pt   = tlv_HHbregrsvfit.Pt();
+				  theSmallTree.m_HHbregrsvfit_eta  = tlv_HHbregrsvfit.Eta();
+				  theSmallTree.m_HHbregrsvfit_phi  = tlv_HHbregrsvfit.Phi();
+				  theSmallTree.m_HHbregrsvfit_m    = tlv_HHbregrsvfit.M();
 			    
+				  std::map<std::string, std::vector<float>> result = ParticleNet_SF(tlv_fj.Pt(), PERIOD);
+				  // Set scale factors for different keys
+				  setScaleFactor(result, "HP",
+								 theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF,
+								 theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_up,
+								 theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_down);
+				  setScaleFactor(result, "MP",
+								 theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF,
+								 theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_up,
+								 theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_down);
+				  setScaleFactor(result, "LP",
+								 theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF,
+								 theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_up,
+								 theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_down);
 			    
-			  // saving infos for subjets
-			  if (theBigTree.ak8jets_nsubjets->at(fjIdx) >= 2 ) 
-			    {
-			      TLorentzVector tlv_subj1;
-			      TLorentzVector tlv_subj2;
-			      vector<int> sjIdxs = findSubjetIdxs(fjIdx, theBigTree);
+				  // saving infos for subjets
+				  if (theBigTree.ak8jets_nsubjets->at(fjIdx) >= 2) 
```

Comment:
I suppose that with the new boosted category definition, we could remove quite a bit of info regarding subjets from the skims (for another time maybe)